ste/JobPartPlan.go

```diff
@@ -17,10 +17,13 @@ import (
 const DataSchemaVersion common.Version = 6
 
 const (
-	ContentTypeMaxBytes     = 256  // If > 65536, then jobPartPlanBlobData's ContentTypeLength's type  field must change
-	ContentEncodingMaxBytes = 256  // If > 65536, then jobPartPlanBlobData's ContentEncodingLength's type  field must change
-	MetadataMaxBytes        = 1000 // If > 65536, then jobPartPlanBlobData's MetadataLength field's type must change
-	BlobTierMaxBytes        = 10
+	ContentTypeMaxBytes        = 256  // If > 65536, then jobPartPlanBlobData's ContentTypeLength's type  field must change
```

Comment:
minor:: maybe just use one variable? BlobCustomHeaderMaxBytes or something similar

Reply:
I was mostly following the pattern I had noticed, but I will go ahead and change this.

Reply:
Understandably though, the patterns of the old codebase (as we already know) were not very fantastic